addon/components/labs-bbl-lookup.js

```diff
@@ -58,7 +58,7 @@ export default Component.extend({
       const validLot = this.get('validLot');
 
       if (validBlock && !validLot) {
-        const SQL = `SELECT the_geom FROM dof_dtm_block_centroids WHERE block= ${parseInt(block, 10)} AND borocode = '${code}'`;
```

Comment:
@bmarchena I see that you closed it out. Do you still need this change to the query string for `dof_dtm_block_centroids`